tests/foreman/api/test_template.py

```diff
@@ -106,7 +106,7 @@ def tftpboot(module_org):
         },
         'ipxe': {
             'setting': 'global_iPXE',
-            'path': f'https://{settings.server.hostname}/unattended/iPXE?bootsrap=1',
+            'path': f'{default_sat.url}/unattended/iPXE?bootsrap=1',
```

Comment:
```suggestion
            'path': f'{default_sat.url}/unattended/iPXE?bootstrap=1',
```
bootsrap  > bootstrap ?

Reply:
I think I'd like to leave that out of these commits, because its spelled in master as `bootsrap` right now. I'd happily merge a PR for that and rebase here.

Reply:
Great catch btw!

Reply:
thanks Stephen. It seems to be a typo in the test and it can be corrected. - the GET request does not influence the (currently existing) tests in any way, since we replace the template by our own and do not work with the parameter in any way. That's also why it works even with the typo in place.